README.md

```diff
@@ -12,319 +12,47 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
-md protobuf_build
-cd protobuf_build
-call cmake %root_dir%\protobuf\cmake -G "Ninja" ^
-   -DCMAKE_INSTALL_PREFIX="%root_dir%\protobuf_install" ^
-   -DCMAKE_BUILD_TYPE=Release ^
-   -Dprotobuf_BUILD_EXAMPLES=OFF ^
-   -Dprotobuf_BUILD_SHARED_LIBS=OFF ^
-   -Dprotobuf_BUILD_TESTS=OFF ^
-   -Dprotobuf_MSVC_STATIC_RUNTIME=OFF ^
-   -Dprotobuf_WITH_ZLIB=OFF
-
-call cmake --build . --config Release
-call cmake --build . --config Release --target install
-```
-
-Before running CMake for onnx-mlir, ensure that the bin directory to this protobuf is before any others in your PATH:
-```shell
-set PATH=%root_dir%\protobuf_install\bin;%PATH%
-```
-
-#### MLIR
-Install MLIR (as a part of LLVM-Project):
-
-[same-as-file]: <> (utils/clone-mlir.sh)
-```shell
-git clone https://github.com/llvm/llvm-project.git
-# Check out a specific branch that is known to work with ONNX MLIR.
-cd llvm-project && git checkout 0bf230d4220660af8b2667506f8905df2f716bdf && cd ..
-```
-
-[same-as-file]: <> (utils/build-mlir.cmd)
-```shell
-set root_dir=%cd%
-md llvm-project\build
-cd llvm-project\build
-call cmake %root_dir%\llvm-project\llvm -G "Ninja" ^
-   -DCMAKE_INSTALL_PREFIX="%root_dir%\llvm-project\build\install" ^
-   -DLLVM_ENABLE_PROJECTS=mlir ^
-   -DLLVM_TARGETS_TO_BUILD="host" ^
-   -DCMAKE_BUILD_TYPE=Release ^
-   -DLLVM_ENABLE_ASSERTIONS=ON ^
-   -DLLVM_ENABLE_RTTI=ON ^
-   -DLLVM_ENABLE_ZLIB=OFF ^
-   -DLLVM_INSTALL_UTILS=ON
-
-call cmake --build . --config Release
-call cmake --build . --config Release --target install
-call cmake --build . --config Release --target check-mlir
-```
-
-#### ONNX-MLIR (this project)
-The following environment variables can be set before building onnx-mlir (or alternatively, they need to be passed as CMake variables):
-- MLIR_DIR should point to the mlir cmake module inside an llvm-project build or install directory (e.g., c:/repos/llvm-project/build/lib/cmake/mlir).
-
-This project uses lit ([LLVM's Integrated Tester](http://llvm.org/docs/CommandGuide/lit.html)) for unit tests. When running CMake, we can specify the path to the lit tool from LLVM using the LLVM_EXTERNAL_LIT define, as in the example below. If MLIR_DIR points to an install directory of llvm-project, LLVM_EXTERNAL_LIT is required and %lit_path% should point to a valid lit. It is not required if MLIR_DIR points to a build directory of llvm-project, which will contain lit.
 
-To build ONNX MLIR, use the following commands:
+Help to update the prerequisite are found [here](docs/Prerequisite.md).
 
-[same-as-file]: <> ({"ref": "utils/build-onnx-mlir.cmd", "skip-doc": 2})
-```shell
-git clone --recursive https://github.com/onnx/onnx-mlir.git
-
-set root_dir=%cd%
+At any point in time, ONNX MLIR depends on a specific commit of the LLVM project that has been shown to work with the project. 
+Periodically the maintainers need to move to a more recent LLVM level. 
+Among other things, this requires that the commit string in (utils/clone-mlir.sh) be updated. 
```

Comment:
"be updated." -> "is updated."  ?

Reply:
Changed to "Among other things, this requires to update the commit string in (utils/clone-mlir.sh). "